app/Domains/Redis/Jobs/StoreMetricsInsightsJob.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Domains\Redis\Jobs;
+
+use App\Data\Collections\MediaCollection;
+use App\Data\Models\Metrics;
+use App\Data\Repositories\InsightsRepository;
+use Lucid\Units\Job;
+
+class StoreMetricsInsightsJob extends Job
+{
+    /**
+     * @var string
+     */
+    private $platform;
```

Comment:
in PHP 7.4 properties can have types, and that way we can get rid of `@var` annotation.